Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/FilmGrain.azsl

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <scenesrg.srgi>
+
+ShaderResourceGroup PassSrg : SRG_PerPass
+{
+    Texture2D<float4> m_inputColor;
+    RWTexture2D<float4> m_outputColor;
+    Texture2D<float4> m_grain;
+
+    Sampler m_sampler
+    {
+        AddressU = Mirror;
+        AddressV = Mirror;
+        AddressW = Mirror;
+    };
+
+    // Must match the struct in FilmGrainPass.cpp
+    struct Constants
+    {
+        uint2 m_outputSize; // texture size of output
+        uint2 m_grainTextureSize; // texture size of grain noise
+        float m_intensity; // intensity of effect (0 to 1)
+        float m_luminanceDampening; // factor for dampening effect in areas of high and low luminance
+        float m_tilingScale; // scaling factor for tiling
```

Comment:
Recommend adding this at the end of the struct (after m_tilingScale) for safety. DX12 rules mean that structs need to end at 16 byte boundary
[[pad_to(16)]]